hw/bsp/ek-tm4c123gxl/board.mk

```diff
@@ -0,0 +1,44 @@
+CFLAGS += \
```

Comment:
Add `DEPS_SUBMODULES = hw/mcu/ti` here to fix ci. I have opyimize ci speed by only checkout required module to save time.

https://github.com/hathach/tinyusb/blob/2adb7e719316b12c53a907153cfa0056db1abd70/hw/bsp/ea4357/board.mk#L1